chart_review/commands/ids.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import csv
+import sys
+
+from chart_review import cli_utils
+
+
+def make_subparser(parser: argparse.ArgumentParser) -> None:
+    cli_utils.add_project_args(parser)
+    parser.set_defaults(func=print_ids)
```

Comment:
re" this and labels, maybe not now, but as you add in some of the other commands, should this move to a centralized location/base class?

Reply:
My plan was to let each command own its own specific logic, but provide all the shared code via `cli_utils.py` or other places.

Like you see above with `cli_utils.add_project_args`, but more of that.